scripts/get-cluster-settings

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+source env-prep
+PERSIST=${PERSIST:-"transient"}
+oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query=_cluster/settings?include_defaults=true -XGET | python -m json.tool
```

Comment:
The pipe to python is for formatting?  You could also do: 
```
oc exec -n $LOGGING_NS -c elasticsearch $pod -- es_util --query='_cluster/settings?include_defaults=true&pretty'
```
Note the tic marks in query